paddle/math/BaseMatrix.h

```diff
@@ -457,6 +457,13 @@ class BaseMatrixT {
 
   /**
    * @code
+   * swap(this, b)
```

Comment:
We need to describe how are we going to use `deepswap` in this usage comment, and make sure that the example code comes into our auto-generated API documentation.